tests/python/frontend/paddlepaddle/test_forward.py

```diff
@@ -552,6 +598,7 @@ def forward(self, inputs):
         verify_model(Conv2DTranspose(stride=2, padding="SAME", dilation=1), input_data=input_data)
         verify_model(Conv2DTranspose(stride=2, padding=3), input_data=input_data)
         verify_model(Conv2DTranspose(stride=3, padding="SAME", groups=1), input_data=input_data)
+        verify_model(Conv2DTranspose(stride=3, padding="SAME", groups=1, data_layout='NHWC'), input_data=input_data)
```

Comment:
Is this supposed to point to the newly introduced class `Conv2DTranspose1` instead?

Reply:
Done.